newa/cli.py

```diff
@@ -844,7 +845,7 @@ def cmd_execute(ctx: CLIContext, workers: int, _continue: bool) -> None:
             launch_description += '<br><br>'
         # add the number of jobs
         if not jira_id.startswith(JIRA_NONE_ID):
-            launch_description += f'{jira_id}: '
+            launch_description += f'[{jira_id}]({jira_url}/browse/{jira_id}): '
```

Comment:
Could you maybe import `urllib` and use
```
issue_url = rllib.parse.urljoin(
                jira_url,
                f"/browse/{jira_id}")
launch_description += f'[{jira_id}]({issue_url}): '
```
? In the code we tried to avoid building URLs like this.

Reply:
OK, I updated it and used `urllib`.